src/platform/Linux/ThreadStackManagerImpl.cpp

```diff
@@ -685,15 +685,187 @@ CHIP_ERROR ThreadStackManagerImpl::_WriteThreadNetworkDiagnosticAttributeToTlv(A
     case ThreadNetworkDiagnostics::Attributes::RouteTableList::Id:
     case ThreadNetworkDiagnostics::Attributes::SecurityPolicy::Id:
     case ThreadNetworkDiagnostics::Attributes::OperationalDatasetComponents::Id:
-    case ThreadNetworkDiagnostics::Attributes::ActiveNetworkFaultsList::Id: {
+    case ThreadNetworkDiagnostics::Attributes::ActiveNetworkFaultsList::Id:
         err = encoder.EncodeEmptyList();
         break;
-    }
-    default: {
+    case ThreadNetworkDiagnostics::Attributes::Channel::Id:
+        err = encoder.Encode(static_cast<uint16_t>(0));
```

Comment:
Is that a valid value?  Same for the other values here....

Note also https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/5195